SCons/Tool/msvc.py

```diff
@@ -236,7 +256,8 @@ def generate(env):
         shared_obj.add_emitter(suffix, shared_object_emitter)
 
     env['CCPDBFLAGS'] = SCons.Util.CLVar(['${(PDB and "/Z7") or ""}'])
-    env['CCPCHFLAGS'] = SCons.Util.CLVar(['${(PCH and "/Yu%s \\\"/Fp%s\\\""%(PCHSTOP or "",File(PCH))) or ""}'])
```

Comment:
its odd that the subst here does `File(PCH)` which will append the current sconscript dir if the PCH is a string (but does not if PCH is already a File object). Seems to be to support the use case in hierarchal.py test.